### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

**Review by CalMacCQ — APPROVED** [review]

**Comment by cqc-melf** [comment]
Will update with PR again after the pytket release is out.

**Comment by cqc-melf** [comment]
Should include all the changes now including the next release. Github secrets should be updated, too.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,84 @@
...
+
+`pytket-cirq` is available for Python 3.8, 3.9 and 3.10, on Linux, MacOS
+and Windows. To install, run:
+
+```pip install pytket-cirq```
+
+This will install `pytket` if it isn't already installed, and add new classes
+and methods into the `pytket.extensions` namespace.
+
+The [example notebooks](https://github.com/CQCL/pytket-cirq/tree/develop/examples) also demonstrate some of the key features of pytket-cirq.
```
Comment (cqc-alec) [inline README.md:26]: This folder does not exist.

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,84 @@
...
+
+
 # pytket-cirq
-pytket-cirq, extensions for pytket quantum SDK 
+
+[Pytket](https://cqcl.github.io/tket/pytket/api/index.html) is a python module for interfacing
+with tket, a quantum computing toolkit and optimisation compiler developed by Quantinuum.
+
+`pytket-cirq` is an extension to `pytket` that allows `pytket` circuits to
+be executed on Quantinuum's quantum devices.
```
[the remark this answers is not in the document]
Comment (cqc-melf) [reply README.md]: Thank you, I now made a checklist with all your comments from the PR and the AQT one to check for the other first before I ask for the review, hope I will see more of this error next time myself.

File: README.md (hunk 1 above)
Comment (cqc-melf) [reply README.md]: removed, see above

File: .github/workflows/build-test [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+# Usage:
+#
+# build-test [mypy|nomypy]
+#
+# Arguments:
+# - mypy: include mypy check ("mypy" or "nomypy")
+#
+# Environment variables used:
+# - MODULE: single module to test
```
[the remark this answers is not in the document]
Comment (cqc-alec) [reply .github/workflows/build-test]: But GITHUB_WORKSPACE is still used, so that should be left in (just MODULE is no longer used).

**Comment by cqc-melf** [comment]
Oh, will add that to my checklist, too ... :( tought I had done that...

**Pull request merged**